config/initializers/devise.rb

```diff
@@ -0,0 +1,14 @@
+Devise.setup do |config|
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
Haha, nice one! :-) I thought there is no need to change it right now since it will still be a fake address instead of a proper one, just a different one. Should I change it anyway? Btw, maybe this was just a joke and I'm being fooled here? ;-)